src/components/index.jsx

```diff
@@ -1,6 +1,7 @@
+import ActionBtn from "./ActionBtn/ActionBtn";
 import Header from "./Header/Header";
 import Sidebar from "./Sidebar/Sidebar";
 import Toast from "./Toast/Toast";
 import VideoCard from "./VideoCard/VideoCard";
 
-export { Toast, Header, VideoCard, Sidebar };
+export { Toast, Header, VideoCard, Sidebar, ActionBtn };
```

Comment:
no need to this line you can replace with import to export

Reply:
Thank You. Will do that during refactoring.